source/features/sticky-sidebar.css

```diff
@@ -15,8 +18,7 @@
 	top: 76px !important; /* Sticky header's height + sidebar item's margin */
 
 	/* Needed so the backdrop of notification modal is over the other elements in the conversation */
-	/* Higher than sticky readme header #4192 */
-	z-index: 90;
+	z-index: 1;
```

Comment:
It seems this is the lowest we can go. 90 was only needed on the readme. Last change was in https://github.com/sindresorhus/refined-github/pull/4364